modules/rtpengine/rtpengine.c

```diff
@@ -1804,8 +1804,11 @@ static int parse_flags(struct ng_flags_parse *ng_flags, struct sip_msg *msg,
 					err = "missing value";
 					if (!val.s)
 						goto error;
-				}
-				break;
+				} else if (str_eq(&key, "received-from")) {
+					if (val.s)
+						ng_flags->received_from = val;
+				} else
+					break;
```

Comment:
Missing `continue`?

Reply:
Yup, will push a fix now.

Reply:
This was actually already fixed by Nick.

Reply:
P.S. You guys have gone missing from freenode. Is there a new secret hideout where all the cool kids are? ;-)